api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/FeeVersionDto.java

```diff
@@ -78,4 +78,7 @@ public BigDecimal getAmount() {
     @JsonProperty("direction")
     private String direction;
 
+    @JsonProperty("reason_for_update")
+    private String reasonForUpdate;
```

Comment:
Shouldn’t we need a Null/Empty validation on the reason of Fee Version Change?
Please confirm if UI needs to throw a Business Validation Message from Backend API

Reply:
Frontend, Null/Empty validation is there ... but anyway, added NotNull as it should be mandatory from backend perspective also... I don't think message from backend is required.